### DISCUSSION TITLE
Can i load Connector on the Modrinth app

### DISCUSSION THREAD

**Discussion by 0NoShow0** [Sinytra/Connector#1304] [Q&A] [answered]
i know this may be a stupid question to ask but I'm trying to put together a modpack on modrinth and i was wondering if i could download Connector and put it in my mods folder and it will load in the modrinth app.

**Comment by GitHub-source-42** [comment, marked as answer]
https://modrinth.com/mod/connector 
it is already hosted on Modrinth

**Reply by 0NoShow0** [reply]
thank you and sorry for the trouble
